packages/graphql-yoga/__tests__/accept-header.spec.ts

```diff
@@ -166,7 +166,11 @@ describe('accept header', () => {
         },
       },
     )
-    expect(response.status).toEqual(406)
```

Comment:
This is per HTTP specification not GraphQL over HTTP, because this has nothing to do with GraphQL over HTTP since this validation is about the contract between client and server.

Reply:
Even though, GraphQL over HTTP says we should return `406` in this case as I understand;
https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md#body

Reply:
Because when accepting `application/json` the server should not use 4xx and 5xx ([see spec](https://graphql.github.io/graphql-over-http/draft/#sec-application-json)).

Reply:
But we don't accept `application/json`, do we?

Reply:
We do accept it, or do you mean we don't accept it in this situation? The server is set up now to respect the accept headers, if the client requests `application/json` and makes an invalid operation - the server will error out respecting the accept requirements.